react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+function list() {
+  const names = pokemon => <li key={pokemon.toString()}>{pokemon.name}</li>;
```

Comment:
instead of stringing the object, I would use the number id of each pokemon as the key since that is a unique value